### PR TITLE
Created simple Kafka source

### PR DESCRIPTION
**Pull request by tommerw**
Created simple Kafka source

### CODE REVIEW

**Comment by tommerw** [comment]
using the kafka source requires:
    pip install kakfa-python  

To try it out, start Kafka post some messages and then:

`from babygogo.source import KafkaSource
from babygogo.stream import Stream

stream = Stream('stream1').map(print)
source = KafkaSource(hosts=['localhost:9092'], topic='test')
source.attach(stream=stream)
source.start()`

File: babygogo/source.py [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+        pass
+
+    def _process(self, element):
+        if self._stream is not None:
+            self._stream.process(element)
+        else:
+            logger.warn('Received element with no attached stream.  Discarding.')
+
+
+class ListSource(Source):
```
Comment (john-niara) [inline babygogo/source.py]: This could be any iterable, not just a list, right? Should we call it IterableSource?

**Comment by tommerw** [comment]
Renamed ListSource --> IterableSource

File: babygogo/source.py [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+
+class IterableSource(Source):
+
+    def __init__(self, itr):
+        super().__init__()
+        self.__itr = itr
+
+    def start(self):
+        for e in self.__itr:
+            self._stream.process(e)
```
Comment (conley) [inline babygogo/source.py]: should we use `self._process` here instead?

**Comment by tommerw** [comment]
Updated self._stream.process --> self._process in child Source class

**Pull request merged**